content/docs/api-reference/dvcfilesystem.md

```diff
@@ -45,11 +45,17 @@ can also specify `encoding` argument in case of text mode (`"r"`).
 ## Reading a file
 
 ```py
->>> contents = fs.cat_file("get-started/data.xml")
+>>> text = fs.read_text("get-started/data.xml", encoding="utf-8")
```

Comment:
💅🏼 not sure we want these `>>>`s.
```suggestion
text = fs.read_text("get-started/data.xml", encoding="utf-8")
```

Reply:
> Other general feedback on the doc: https://github.com/iterative/dvc.org/pull/3932#pullrequestreview-1145171639

Reply:
So can we remove the `>>>` for consistency? Also, they look like REPL but this could apply to any Py env.

Reply:
We can discuss about `>>>` separately.

Reply:
I want to show the output together with the code, the way you get in the REPL.

Reply:
> show the output together with the code

p.s. I think we would use the `cli` highlighter for this.

Reply:
Also, you wouldn't see code in REPL (terminal) mixed with comments, right?

<img width="725" alt="image" src="https://user-images.githubusercontent.com/1477535/203178214-33b491db-7445-4c25-acee-73e080ca22ec.png">